app/app.go

```diff
@@ -491,6 +496,12 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	// note replicate if you do not need to test core IBC or light clients.
 	app.ScopedIBCMockKeeper = scopedIBCMockKeeper
 
+	// TODO: update height once proposal is passed
+	app.AddPatch(3038452, func(ctx sdk.Context) error {
+		app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
+		return nil
+	})
```

Comment:
so, this is a hack to get consensus breaking change without doing a `SoftwareUpgrade` proposal?

Reply:
Yes, correct